package.json

```diff
@@ -21,11 +21,11 @@
     "changelog:update": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:update",
     "changelog:validate": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:validate",
     "create-package": "ts-node scripts/create-package",
-    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn constraints && yarn lint:dependencies && yarn lint:teams",
+    "lint": "yarn lint:eslint && echo && yarn lint:misc --check && yarn constraints && yarn lint:dependencies && yarn lint:teams",
```

Comment:
What is this echo for?

Reply:
It's to add a buffer between the output of `yarn lint:eslint` and the output of `yarn lint:misc`. The last line in `yarn lint:eslint` may be an error and I didn't want people to miss it. I did add some color to the error, so that certainly helps, but without some space the output still looks a bit messy and I thought it might help readability to add it.

Reply:
Ah I see, OK, makes sense.